hack/run-kiali.sh

```diff
@@ -96,6 +96,7 @@ while [[ $# -gt 0 ]]; do
     -ce|--client-exe)            CLIENT_EXE="$2";                    shift;shift ;;
     -es|--enable-server)         ENABLE_SERVER="$2";                 shift;shift ;;
     -gu|--grafana-url)           GRAFANA_URL="$2";                   shift;shift ;;
+    -iu|--istiod-url)            ISTIOD_URL="$2";                    shift;shift ;;
     -in|--istio-namespace)       ISTIO_NAMESPACE="$2";               shift;shift ;;
```

Comment:
Options are in alphabetical order - so swap these around:
```suggestion
    -in|--istio-namespace)       ISTIO_NAMESPACE="$2";               shift;shift ;;
    -iu|--istiod-url)            ISTIOD_URL="$2";                    shift;shift ;;
```

Reply:
Note that you need to add a help description for this new option down on line 155. Something like:
```
  -iu|--istiod-url
      The URL of the istiod endpoint.
      Default: http://127.0.0.1:15014/version
```

Note that the defaults are normally defined via a "DEFAULTS" env var so you don't duplicate the values in multiple areas in the script. So you should consider adding that and then just using it here (`Default: ${DEFAULT_ISTIOD_URL}`) and down below where that default value is set.

Reply:
You also need to add this new setting to the configuration file, otherwise, it isn't really being used. So add this section

```
istio:
  url_service_version: "${ISTIOD_URL}"
```

to this file after the grafana section (to retain alphabetic sorting of the settings): https://github.com/kiali/kiali/blob/master/hack/run-kiali-config-template.yaml#L32

Here's the diff that I just used to test this and it works:

```diff
diff --git a/hack/run-kiali-config-template.yaml b/hack/run-kiali-config-template.yaml
index 4fa6bf601..242d9dedd 100644
--- a/hack/run-kiali-config-template.yaml
+++ b/hack/run-kiali-config-template.yaml
@@ -30,6 +30,9 @@ external_services:
     in_cluster_url: "${GRAFANA_URL}"
     url: "${GRAFANA_URL}"
 
+  istio:
+    url_service_version: "${ISTIOD_URL}"
+
   prometheus:
     url: "${PROMETHEUS_URL}"
```

Reply:
Thanks, @jmazzitelli 
I didn't know about that disease ^^
But is pretty helpful, especially when you have so many options.